gradoop-flink/src/main/java/org/gradoop/flink/io/impl/csv/metadata/CSVMetaDataSink.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright © 2014 - 2019 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.flink.io.impl.csv.metadata;
+
+import org.apache.flink.api.java.DataSet;
+import org.apache.flink.api.java.tuple.Tuple3;
+import org.apache.flink.core.fs.FileSystem;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FSDataOutputStream;
+import org.apache.hadoop.fs.Path;
+import org.gradoop.common.model.impl.metadata.PropertyMetaData;
+import org.gradoop.flink.io.api.metadata.MetaDataSink;
+import org.gradoop.flink.io.api.metadata.MetaDataSource;
+import org.gradoop.flink.io.impl.csv.CSVConstants;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Implementaion of a {@link MetaDataSink}
+ */
+public class CSVMetaDataSink implements MetaDataSink<CSVMetaData> {
+
+  @Override
+  public void writeDistributed(
+    String metaDataPath,
+    DataSet<Tuple3<String, String, String>> metaDataTuples,
+    FileSystem.WriteMode writeMode) {
+    metaDataTuples.writeAsCsv(metaDataPath, CSVConstants.ROW_DELIMITER,
+      CSVConstants.TOKEN_DELIMITER, writeMode).setParallelism(1);
+
+  }
+
+  @Override
+  public void writeLocally(
```

Comment:
`writeLocal` sounds better in my opinion.

Reply:
Done!